taipy/gui/gui.py

```diff
@@ -369,6 +369,8 @@ def __init__(
         The returned HTML content can therefore use both the variables stored in the *state*
         and the parameters provided in the call to `get_user_content_url()^`.
         """
+        # Notification callbacks
+        self._notification_callbacks = {}
```

Comment:
please remove this
the callback name should be sent to the front-end and come back from the front-end